core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -422,8 +422,13 @@ private[spark] object UIUtils extends Logging {
    * the whole string will rendered as a simple escaped text.
    *
    * Note: In terms of security, only anchor tags with root relative links are supported. So any
-   * attempts to embed links outside Spark UI, or other tags like &lt;script&gt; will cause in
-   * the whole description to be treated as plain text.
+   * attempts to embed links outside Spark UI, or other tags like
+   *
+   * {{{
+   * <script>
+   * }}}
+   *
+   * will cause in the whole description to be treated as plain text.
```

Comment:
- Java
  <img width="940" alt="2016-11-27 3 19 01" src="https://cloud.githubusercontent.com/assets/6477701/20642369/57ee89b6-b450-11e6-83e4-42fd3737d794.png">


Scaladoc not found